### PR TITLE
Mziabari blitz doc updated

### PR DESCRIPTION
**Pull request by kamyarz1991**

### CODE REVIEW

File: develop-guide/writeblitz/writeblitz.rst [hunk 1]
```diff
@@ -0,0 +1,759 @@
+.. _write-blitz:
+
+Write a BLITZ
```
Comment (jeaubin) [inline develop-guide/writeblitz/writeblitz.rst]:
```
Quick Trigger (Blitz)
--------------------
```

File: develop-guide/writeblitz/writeblitz.rst [hunk 2]
```diff
@@ -0,0 +1,759 @@
...
+called *BLITZ* because it's lightning fast --- does the following actions:
+
+* Configure a device.
+* Parse the device output to verify if the device state is as expected.
+* Unconfig or modify the initial configuration.
+* Parse the output to check the operational state.
+* Learn a feature and verify the result of the action
+* Calling different apis and use their outputs on other actions and other devices
+* Yang integration
+* It is fully customizable and new actions can be added
```
Comment (jeaubin) [inline develop-guide/writeblitz/writeblitz.rst]: add a Many more, the full list would be below

File: develop-guide/writeblitz/writeblitz.rst [hunk 3]
```diff
@@ -0,0 +1,759 @@
...
+              # the bgp protocol state is shutdown
+              - parse:
+                  device: R3_nx
+                  # All action supports banner field to add to the log
+                  banner: Verify bgp process is shutdown
+                  command: show bgp process vrf all
+                  include:
+                    - get_values('shutdown')
+                  exclude:
+                    - not_contains('bgp_protocol_state')
```
Comment (jeaubin) [inline develop-guide/writeblitz/writeblitz.rst]: this should be running

File: develop-guide/writeblitz/writeblitz.rst [hunk 4]
```diff
@@ -0,0 +1,759 @@
...
+        continue: True
+        function: get_interface_mtu_size
+        arguments:
+            interface: GigabitEthernet1
+        include:
+            - 1500
+        exclude:
+            - 9999
+
+Replying to the prompt dialouge
```
Comment (jeaubin) [inline develop-guide/writeblitz/writeblitz.rst]: Typo in dialogue

Comment (KamyarZiabari) [reply develop-guide/writeblitz/writeblitz.rst]: Dome

File: develop-guide/writeblitz/writeblitz.rst (hunk 2 above)
Comment (KamyarZiabari) [reply develop-guide/writeblitz/writeblitz.rst]: Done

File: develop-guide/writeblitz/writeblitz.rst (hunk 1 above)
Comment (KamyarZiabari) [reply develop-guide/writeblitz/writeblitz.rst]: Done

**Pull request merged**